app/controllers/admin_controller.rb

```diff
@@ -402,6 +402,36 @@ def user_search
 
   end
 
+  def settings
+    @settings = AppSettings.get_all
+
+    respond_to do |format|
+      format.html
+    end
+
+  end
+
+  def update_settings
+
+    @settings = AppSettings.get_all
+
+    # iterate through
+    @settings.each do |setting|
+#      logger.info("Setting: #{setting[0]}")
+#      logger.info("Value: #{params[setting[0]]}")
+
+      AppSettings[setting[0]] = params[setting[0].to_sym]
```

Comment:
I am think ```params``` hash works nice with strings and symbols as keys, why to use `to_sym` here ?